app-website-4/ProductsApi/Startup.cs

```diff
@@ -20,6 +24,21 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddSingleton<ProductStore>();
+            services.AddSingleton<TokenProvider>();
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
+            {
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ValidateIssuer = true,
+                    ValidateAudience = true,
+                    ValidateIssuerSigningKey = true,
+                    ValidIssuer = "Products API",
+                    ValidAudience = "Products App",
+                    IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes("This is a secret key"))
```

Comment:
would make use of that service returning the key as mentioned earlier here

Reply:
👍